ios/RNSBarButtonItem.h

```diff
@@ -0,0 +1,12 @@
+#import <UIKit/UIKit.h>
+
+typedef void (^RNSBarButtonItemAction)(NSString *buttonId);
+typedef void (^RNSBarButtonMenuItemAction)(NSString *menuId);
+
+@interface RNSBarButtonItem : UIBarButtonItem
+
+- (instancetype)initWithDictionary:(NSDictionary<NSString *, id> *)dict
```

Comment:
```suggestion
- (instancetype)initWithConfig:(NSDictionary<NSString *, id> *)dict
```

Reading this code for the first time, I've had no idea what's inside this "dictionary". Let's use the parameter description to clarify that.